### ISSUE TITLE
Steam Cloud sync sometimes does not work

### ISSUE THREAD

**Issue by SteffenL** [SteffenL/dise#17]
I have been informed that Steam Cloud syncing sometimes fails with the error message below. Saving repeatedly can make it work.

![AexRnpr](https://user-images.githubusercontent.com/1543854/58744848-90997a80-8448-11e9-9ca5-9dcdf3a71e33.png)

**Comment by SteffenL** [comment]
I have noticed this happen a few more times. When this happened to me, I also noticed that Steam starting to sync (download) other files in the same directory at the same time. I wonder if there is some sort of timing problem which makes Steam lock the file just as DISE attempts to rewrite it. In this case, DISE could retry a few times with short delays. That would probably be a good enough solution.
